### DISCUSSION TITLE
Should this library be discontinued?

### DISCUSSION THREAD

**Discussion by roycornelissen** [roycornelissen/GMImagePicker.Xamarin#86] [Q&A]
I've been looking into recent issues with this library on iOS 14, like #80 and #83. In iOS 14, Apple introduced the new [Photos Picker](https://developer.apple.com/videos/play/wwdc2020/10652/) (`PHPicker`), which seems to cover some of the important features that `GMImagePicker.Xamarin` added to the old default `UIImagePicker` (most notably multi-selection). Moreover, `PHPicker` deals with the updated security model around the media library.

I'm starting to feel like this library has had its time, and that `PHPicker` should be preferred over `GMImagePicker` in iOS 14.

Just wanted to throw this out here, and see what the community thinks. Does it make sense to keep this library around? If so, it will need an overhaul to comply with the updated security model since iOS 14, and I'd love to get some help on this, since the available time I can spend on this library has diminished quite a bit.

**Comment by lazmeister** [comment]
Hi Roy, The only issue is from what I can tell is that PHPicker is only supported iOS 14+ (not confirmed).  Matthew has created a great plugin for PHPicker here: https://github.com/MatthewKapteyn/MediaPlugin/tree/master/src/Media.Plugin/iOS  (tested older iOS versions with issues)

**Reply by giuseppenovielli** [reply]
@lazmeister @roycornelissen  i launch  PHPicker into iOS 11.0 and throw an exception

> System.Exception
> Could not create an native instance of the type 'PhotosUI.PHPickerConfiguration': the native class hasn't been loaded.
> It is possible to ignore this condition by setting ObjCRuntime.Class.ThrowOnInitFailure to false.

So must to continue use this library to support iOS <14, mostly for Multiple Selection, which UIImagePickerViewController not supported!

**Comment by rr-rv** [comment]
I like this lib, and would love to see it updated, how ever i cant offer any help my self. 

If you end up letting it slide i just want to thank you for making this in the first place!

**Reply by burhan1875** [reply]
Is a new version available in the meantime or is it not worth waiting for? Thanks
